### PR TITLE
[WIP] SITES-447 Add "popular now" hard coded links to the top of the home page

### PR DESCRIPTION
**Pull request by maxious**
waiting on final content....
screen shot 2016-07-27 at 11 08 11 am

### CODE REVIEW

**Comment by tessereth** [comment]
LGTM

**Comment by simonschwartz** [comment]
The UI Kit team have removed margin-top from headings so just needed to add some space between the links and the heading when everything stacks on mobile.

LGTM

**Comment by micapam** [comment]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ This file is influenced by http://keepachangelog.com/.
 - Added admin-only option to stop a node appearing in breadcrumbs and the menu
 - Added links on department homepages to related ministers
 - Added long summary field for section_home form and template
+- Add "popular now" hard coded links to the top of the home page
```
Comment (tessereth) [inline CHANGELOG.md]: This line will need to be moved up in the changelog now that we've made a release.

**Comment by micapam** [comment]
Hey @maxious, have you seen latest design for these links? I think they are supposed to go in two rows now rather than in a single column ... not sure if that's a pure-CSS change or if it'll need HTML changes? Or even if the UIKit changes are actually ready ... just thought I'd mention it.

**Comment by maxious** [comment]
Yep, hence this PR is blocked on the new CSS/HTML being defined.

**Pull request closed without merging**